continuous_integration/environment-3.10-jdk11-dev.yaml

```diff
@@ -40,4 +40,5 @@ dependencies:
 - qpd
 - triad
 - pip:
-  - fugue[sql]>=0.5.3
+  # TODO: remove max version constraint when fugue adds back python 3.9+ support
```

Comment:
Thanks for confirming. In that case can we reword this to indicate that it's not 3.9+ related

Reply:
Yup! Sorry, forgot to bump this message once I realized the failures were unrelated